src/Orchard.Web/Modules/Orchard.Azure/Services/TaskLease/AzureApplicationEnvironment.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.WindowsAzure.ServiceRuntime;
+using Orchard.Environment;
+
+namespace Orchard.Azure.Services.TaskLease {
```

Comment:
It was generated as such by R# because of its folder structure. OK if I change that too?